app/src/main/res/layout/photo_item_grid.xml

```diff
@@ -17,7 +17,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_alignParentStart="true"
-        android:layout_alignParentTop="true"
+        android:layout_alignParentBottom="true"
```

Comment:
you cant just change something like this, it now fully overlaps with the file name